disnake/guild.py

```diff
@@ -2555,8 +2555,10 @@ async def edit_welcome_screen(
     def fetch_members(
         self, *, limit: Optional[int] = 1000, after: Optional[SnowflakeTime] = None
     ) -> MemberIterator:
-        """Retrieves an :class:`.AsyncIterator` that enables receiving the guild's members. In order to use this,
-        :meth:`Intents.members` must be enabled.
+        """Retrieves an :class:`.AsyncIterator` that enables receiving the guild's members.
+
+        In order to use this, the :attr:`~Intents.members` intent must be
+        enabled in the developer portal.
 
```

Comment:
versionchanged should be added here too IMO

```rst
.. versionchanged:: 2.9
    No longer requires the Intent to be enabled on the websocket connection. 
```

Reply:
https://github.com/DisnakeDev/disnake/pull/1013/commits/49fa9e2ca436a35347d61a28ae3879dd70218981